src/Commands/DrupalCommands.php

```diff
@@ -88,11 +90,13 @@ public function siteInstall(array $options = [
       'account-name'      => InputOption::VALUE_REQUIRED,
       'account-password'  => InputOption::VALUE_REQUIRED,
       'account-mail'      => InputOption::VALUE_REQUIRED,
-      'database-user'     => InputOption::VALUE_REQUIRED,
-      'database-password' => InputOption::VALUE_REQUIRED,
+      'database-type'     => InputOption::VALUE_REQUIRED,
       'database-host'     => InputOption::VALUE_REQUIRED,
       'database-port'     => InputOption::VALUE_REQUIRED,
       'database-name'     => InputOption::VALUE_REQUIRED,
+      'database-user'     => InputOption::VALUE_REQUIRED,
+      'database-password' => InputOption::VALUE_REQUIRED,
+      'database-url'      => InputOption::VALUE_OPTIONAL,
```

Comment:
Let's not provide this option, the individual URL elements already contain all the information we need to construct a valid URL.